src/cmd/root.go

```diff
@@ -36,16 +36,21 @@ func init() {
 	rootCmd.PersistentFlags().String("api-token", "", "The OpsLevel API Token. Overrides environment variable 'OPSLEVEL_API_TOKEN'")
 	rootCmd.PersistentFlags().String("log-format", "TEXT", "overrides environment variable 'OPSLEVEL_LOG_FORMAT' (options [\"JSON\", \"TEXT\"])")
 	rootCmd.PersistentFlags().String("log-level", "INFO", "overrides environment variable 'OPSLEVEL_LOG_LEVEL' (options [\"ERROR\", \"WARN\", \"INFO\", \"DEBUG\"])")
+	rootCmd.PersistentFlags().Bool("scaling-enabled", false, "Enables built-in pod scaling for kubernetes environment, defaults to false for local development")
 
+	// TODO: Update pod-* flags to job-pod-* to distinguish between job and runner specific values.  Moving this work over to separate ticket since it's a breaking change: https://gitlab.com/jklabsinc/OpsLevel/-/issues/5763
 	rootCmd.PersistentFlags().Int("pod-max-wait", 60, "The max amount of time to wait for the job pod to become healthy.")
-	rootCmd.PersistentFlags().String("pod-namespace", "default", "The kubernetes namespace to create pods in.")
+	rootCmd.PersistentFlags().String("pod-namespace", "default", "The kubernetes namespace to create job pods in.")
 	rootCmd.PersistentFlags().Int64("pod-requests-cpu", 1000, "Default is in millicores.")
 	rootCmd.PersistentFlags().Int64("pod-requests-memory", 1024, "Pod job resource requests in MB.")
 	rootCmd.PersistentFlags().Int64("pod-limits-cpu", 1000, "Default is in millicores.")
 	rootCmd.PersistentFlags().Int64("pod-limits-memory", 1024, "Pod job resource limits in MB.")
 	rootCmd.PersistentFlags().String("pod-shell", "/bin/sh", "The shell to use for commands inside the pod.")
 	rootCmd.PersistentFlags().Int("pod-log-max-interval", 30, "The max amount of time between when pod logs are shipped to OpsLevel. Works in tandem with 'pod-log-max-size'")
 	rootCmd.PersistentFlags().Int("pod-log-max-size", 1000000, "The max amount in bytes to buffer before pod logs are shipped to OpsLevel. Works in tandem with 'pod-log-max-interval'")
+	rootCmd.PersistentFlags().String("runner-pod-name", "", "overrides environment variable 'RUNNER_POD_NAME'")
+	rootCmd.PersistentFlags().String("runner-pod-namespace", "default", "The kubernetes namespace the runner pod is deployed in.")
+	rootCmd.PersistentFlags().String("runner-deployment", "runner", "The kubernetes namespace the runner pod is deployed in.")
```

Comment:
add an ENV VAR for this one too?

Reply:
Also the description is incorrect

Reply:
Oh yep, forgot to update it when I copied it over.